deploy/deploy_to_bintray.sh

```diff
@@ -44,16 +44,15 @@ if [[ ! -d "${SOURCE_DIR}" || $(ls -A "${SOURCE_DIR}" | wc -l) -eq 0 ]]; then
     echo "Directory '$SOURCE_DIR' doesn't exist or is empty";
     exit 1;
 fi
-BINTRAY_API_KEY=$2
+
 # for PR's this is set if the PR comes from within the same repository
 if [ "${BINTRAY_API_KEY}" == "" ] ; then
     echo "BINTRAY_API_KEY is missing; doing nothing"
     exit 0
 fi
 
-PULL_REQUEST=$3
-
-CI_RUN="${CI:-false}"
+#CI_RUN="${GITHUB_ACTIONS:-false}"
+CI_RUN="false"
```

Comment:
I believe this is a debug leftover and should be changed back?

Reply:
Yes, this is intentional. This will not make it over to the main repository. For now please keep it in.

Reply:
ok, so I'll note to change it back before merging

Reply:
Hopefully before Monday I'll finish what is left